### ISSUE TITLE
The theme is at the latest version

### ISSUE THREAD

**Issue by pingram3541** [afragen/github-updater#389]
This has happened on 2 different servers this week.  The theme indicates there is an update (and there is of course), but when clicking update now it immediate says the theme is at the latest version and doesn't pull the update from github.  The last time this happened I tried about an hour later and it updated just fine.

**Comment by pingram3541** [comment]
I was able to successfully update once the orange update banner showed overlaying the theme screenshot about 10min later.  Prior to the orange update banner showing, I had clicked on the theme details of which it showed a newer version with the update link in the details but wouldn't actually update as mentioned on my initial post.  So it seems there is something not allowing an immediate update and you have to give it some time until the update banner is actually revealed?  Is this normal and expected behavior?

**Comment by afragen** [comment]
Can you provide a URL so I can test?

**Comment by pingram3541** [comment]
Since I am now up to date there really isn't anything to see.  Let me see if I can build a test environment in which I can duplicate this condition.

I've noticed a few times where the update doesn't happen even though an update is indicated and of course there is indeed an update.  Happened on another site today with **this** actual plugin using the manual method of choosing to re-install from master branch (5.4.0 to 5.4.1).  The new dynamic update method was not showing yet but I knew 5.4.0 was old so I tried to force it and it said it was the latest version.  Went to core wordpress updates area and it then showed the plugin update available and I was able to complete the update properly.  After a few minutes like yesterday.

I'm pretty sure this only occurs because I am trying to perform the updates on older versions of a theme or plugin in which I just recently added the github header info to conveniently be able update them.
